src/iop/demosaic.c

```diff
@@ -2871,7 +2878,8 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
   dt_iop_roi_t roo = *roi_out;
   roo.x = roo.y = 0;
   // roi_out->scale = global scale: (iscale == 1.0, always when demosaic is on)
-
+  gboolean info = (darktable.unmuted & DT_DEBUG_DEMOSAIC);
+  
```

Comment:
I *think* this should be combined with `PERF` since you use info to produce perf informations for demosaic

Reply:
There is not only perf data. But this one should be DT_DEBUG_PERF.